test/ct_test.c

```diff
@@ -124,6 +124,7 @@ static CT_TEST_FIXTURE set_up(const char *const test_case_name)
 static void tear_down(CT_TEST_FIXTURE fixture)
 {
     CTLOG_STORE_free(fixture.ctlog_store);
+    SCT_free(fixture.sct);
```

Comment:
No, because I've only just realised that something like this exists - only the hearbeat test appears to have been using it so far. Let me figure out why on earth we have this thing and get back to you.